### DISCUSSION TITLE
能不能出个Android版

### DISCUSSION THREAD

**Discussion by wekingchen** [messense/aliyundrive-webdav#67] [Q&A] [answered]
可以在手机上运行了，直接用nplayer挂载看片

**Comment by messense** [comment]
Android 那就直接用官方客户端呀，为什么要绕道 webdav...

**Reply by gy8878736** [reply]
官方客户端不支持内封字幕与多声道，也不支持原画hdr，杜比视界等技术。特别是字幕与声道，导致很多资源没法观看！

**Comment by wekingchen** [comment]
因为官方客户端播放体验没有专门的播放器好呀，有了webdav可以有更多的选择~

**Comment by Triwind** [comment, marked as answer]
> 因为官方客户端播放体验没有专门的播放器好呀，有了webdav可以有更多的选择~

https://github.com/eritpchy/webdav-aliyundriver-android

**Comment by Gokou-Ruri** [comment]
你可以下载aarch64 Linux版，用termux就能运行，过程和在Linux上使用是完全一样的。

**Reply by Gokou-Ruri** [reply]
不过缓冲速度不如直接在PC上使用，PC端能跑20M/s，而手机缓冲同一个视频只有5M/s不到。网络环境是联通5G，手机热点共享给电脑，所以可以认为PC和手机的网络状况是相同的，按说手机应该也要跑到和PC相同的缓冲速度才正常。
看了下手机CPU负载基本都在A55小核上面，而且频率也基本处于最低。

测了下发现是MX播放器不支持多线程，用IDM下载器可以到30M/s左右。

@messense 另外我发现使用FX文件管理器可以访问aliyunpan-webdav，但使用[NMM文件管理器](https://play.google.com/store/apps/details?id=in.mfile&hl=zh)却无法访问，进去以后没有文件目录列表，是空白的，等待了很久也没有刷新文件目录。
